index.html

```diff
@@ -248,13 +250,31 @@ <h2>Zoom-control APIs</h2>
     </section>
 
     <section id="scrolling">
-      <h1>Scroll</h1>
+      <h1>Gesture-forwarding</h1>
       <section>
-        <h2>Scrolling APIs</h2>
+        <h2>Definition of GestureForwardingOptions</h2>
+        <pre class="idl">
+          dictionary GestureForwardingOptions {
+            boolean wheel = false;
```

Comment:
Doesn't having the booleans default to false defeat the purpose of having an API that works across platforms like tablets, phones and desktop?

I understand there's a WebIDL issue with booleans defaulting to true, but maybe there's a way to work around that, e.g. enums?